integration/helpers/base_test.py

```diff
@@ -115,9 +116,17 @@ def setUp(self):
         self.output_file_path = None
         self.sub_input_file_path = None
 
+    @retry(
+        stop=stop_after_attempt(3),
+        wait=wait_fixed(30),
```

Comment:
If there is multiple tests running at the same time (assuming so) and they all throttle at the same time, will they throttle again when they wait for 30 seconds? Do we need to add some randomness or exponential to it?

Reply:
I assume the deployment time is different for different tests (not sure if the difference is big enough though).